File: MusicApp/API/Helpers/UserParams.cs
namespace API.Helpers
{
    public class UserParams:PaginationParams
    {
        public string CurrentUsername { get; set; }
        public string InstrumentType { get; set; }
        public string MinCost { get; set; }
        public string MaxCost { get; set; }
        public string OrderBy { get; set; } = "LastApdate";
    }
}

File: MusicApp/client/src/app/models/user-params.ts
import { InstrumentType } from "./instrument-type";
import { User } from "./user";
export class UserParams {
  instrumentType:InstrumentType;
  minCost=0;
  maxCost=20000;
  pageNumber=1;
  pageSize=5;
  orderBy="LastApdate";

  constructor({instrumentType}:User) {
    this.instrumentType=instrumentType;
  }
}
